src/main/resources/assets/botania/lang/zh_CN.lang

```diff
@@ -421,12 +466,12 @@ entity.Botania.botania:manaStorm.name=魔力风暴中心
 entity.Botania.botania:babylonWeapon.name=贮藏库武器
 entity.Botania.botania:fallingStar.name=落星
 
-# ACHIEVEMENT NAMES + DESC (Due to the fact that most achievement names have references to vocaloid song names, please use original name or common name first, thanks.)(以下成就名多为曲名,因此大多使用惯用名)
+# ACHIEVEMENT NAMES + DESC
 achievement.botania:flowerPickup=Sky of Beginning
 achievement.botania:flowerPickup.desc=拾取一朵神秘花
-achievement.botania:lexiconUse=现実的论理主义者
+achievement.botania:lexiconUse=現実的論理主義者
```

Comment:
Nope. They are Japanese Kanji. And yes they looks very similiar to trandtional Chinese script. Same question as you asked before.